.github/workflows/stale.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
           stale-issue-message: 'This issue has been inactive for 14 days. StaleBot will close this stale issue after 14 more days of inactivity.'
-          exempt-issue-labels: 'bug,chore,feature,documentation,testing,operational-excellence,automation,roadmap'
+          exempt-issue-labels: 'bug,chore,feature,documentation,testing,operational-excellence,automation,roadmap,kind/*'
```

Comment:
Have you validated that wildcards work here? Just glancing over the documentation, I don't see it mentioned anywhere that wildcards are supported by this action